docs/docs-content/vertex/system-management/enable-non-fips-settings/use-non-fips-addon-packs.md

```diff
@@ -12,79 +12,76 @@ tags: ["vertex", "non-fips"]
 
 
 Palette VerteX provides the following FIPS-compliant infrastructure components in Kubernetes clusters it deploys. Review [FIPS-Compliant Components](../../fips/fips-compliant-components.md) to learn more.
-
-<br />
-
     
 - Operating System (OS)
-
 - Kubernetes
-
 - Container Network Interface (CNI)
-
 - Container Storage Interface (CSI)
 
-As shown in the screenshot below, the FIPS-compliant icon used to indicate full FIPS compliance is displayed next to Palette VerteX infrastructure components in the cluster profile stack. To learn about other icons Palette VerteX applies, refer to [FIPS Status Icons](../../fips/fips-status-icons.md).
+VerteX provides the flexibility to utilize non-FIPS packs. To do this, you must add a non-FIPS compliant pack registry. Review the [Add a Registry](../add-registry.md) guide. 
+
+We provide an Open Container Initiative (OCI) registry and a community registry. You can add both non-FIPS registries to utilize the entire suite of packs. The table below lists these registries and their endpoints. Our support team will provide you with the required credentials.
+
+| **Production**     | **Type**   | **Endpoint**       | **Base Path**   |
```

Comment:
```suggestion
| **Registry**     | **Type**   | **Endpoint**       | **Base Path**   |
```

I think the term production is a bit confusing in this context. These are the production registries so I think just stating registries makes it easier to understand.  I would also mention that these are hosted on AWS ECR so that they know that.  

Maybe something like this?


|Production | Type | Endpoint | FIPS | Base Path|
-- | -- | -- | --| -- |
OCI Pack Registry | AWS ECR | 415789037893.dkr.ecr.us-east-1.amazonaws.com |  No |  production
Spectro Cloud Community Registry | AWS ECR  | 415789037893.dkr.ecr.us-east-1.amazonaws.com |  No | community